homeassistant/components/shelly/light.py

```diff
@@ -130,46 +134,44 @@ async def async_setup_rpc_entry(
 class BlockShellyLight(ShellyBlockEntity, LightEntity):
     """Entity that controls a light on block based Shelly devices."""
 
+    _attr_supported_color_modes: set[str] = set()
```

Comment:
```suggestion
```
Setting this as class variable it is shared between all instances, I missed it while reviewing, but testing on bulbs I noticed that a white only bulb show RGBW properties :-)

Reply:
I think I need to keep the type hint at the class level, and create the set in the init method.
Can you give that a go please?

Reply:
Looks good, Thanks!